shared/unattended/Ubuntu-18.10-ppc64le.preseed

```diff
@@ -0,0 +1,64 @@
+debconf debconf/priority string critical
+unknown debconf/priority string critical
+d-i debconf/priority string critical
+d-i localechooser/languagelist select en
+d-i debian-installer/locale string en_US
+d-i console-tools/archs select at
+d-i keyboard-configuration/layoutcode string us
+d-i keyboard-configuration/layout select English (US)
+d-i console-setup/ask_detect boolean false
+d-i keyboard-configuration/layoutcode string en
+d-i netcfg/choose_interface select auto
+d-i netcfg/get_hostname string unassigned-hostname
+d-i netcfg/get_domain string unassigned-domain
+d-i netcfg/wireless_wep string
+
+d-i clock-setup/utc boolean true
+d-i time/zone string US/Eastern
+
+d-i partman-auto/method string regular
+d-i partman-auto/choose_recipe select atomic
+d-i partman/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+d-i partman-lvm/device_remove_lvm boolean true
+d-i partman-md/device_remove_md boolean true
+d-i partman-lvm/confirm boolean true
+d-i partman-auto/disk string /dev/sda
+
+d-i passwd/root-login boolean true
+d-i passwd/user-fullname string Ubuntu User
+d-i passwd/username string ubuntu
+d-i passwd/user-password password 123456
```

Comment:
this needs to be matched with default ubuntu passwd `12345678` set in config otherwise, override password to `123456` in 18.10.ppc64le.cfg file.